lib/mayaUsd/python/wrapOpUndoItem.cpp

```diff
@@ -41,38 +41,20 @@ class PythonOpUndoItemList
 
     ~PythonOpUndoItemList() { }
 
-    void enter()
-    {
-        if (!TF_VERIFY(_undoItemList == nullptr)) {
-            return;
-        }
-        _undoItemList = std::make_unique<MayaUsd::OpUndoItemList>();
-        _recorder = std::make_unique<MayaUsd::OpUndoItemRecorder>(*_undoItemList);
-    }
+    void enter() { _recorder = std::make_unique<MayaUsd::OpUndoItemRecorder>(_undoItemList); }
 
-    void exit(object, object, object)
-    {
-        if (!TF_VERIFY(_undoItemList != nullptr)) {
-            return;
-        }
-        _recorder.reset();
-        _undoItemList.reset();
-    }
```

Comment:
Getting rid of the undo item list was wrong: it prevented undo/redo from working at all! That is why the class keeps a permanent undo list now. Makes the code simpler, too.